frontend/components/Vehicles/ConfirmDialog.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <v-dialog v-model="showDialog" max-width="500px">
```

Comment:
`showDialog` is a prop it should not be v-model-ed since props should be immutable.
```suggestion
  <v-dialog :value="show" max-width="500px">
```

Reply:
why should props be immutable? how am i supposed to close this dialog then? what is the difference between value and vmodel in this case?

Reply:
As for the `props` question it's explained in `props` link that I posted in a previous comment 😉 
Difference between `v-model` and `value` is very simple, `v-model` equals to this:
```js
<v-dialog @input="showDialog = $event" :value="showDialog" /> // $event is shorthand, this can be written like this: @input="value => showDialog  = value"
...

data = () => ({ showDialog = false })

```

As for the closing dialog, you can achieve it in various ways. Since you have close/confirm button within the dialog, they will emit the event which in parent sets `showDialog` to false, which will be passed again here. Then `:value` on the dialog will be false and it should be closed. This should do the trick.